spec/features/books/user_can_see_a_book_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+describe "When a user visits a book show page" do
+  scenario "I see the book title and reivews" do
+    book = Book.create(title: "Book1")
+    book.reviews.create(username: "grayton", body: "i really liked this book", rating: 5)
+    book.reviews.create(username: "grayton2", body: "ibook", rating: 3)
+    book.reviews.create(username: "grayton3", body: "i", rating: 1)
```

Comment:
would want to attach a user to a review (saw that you got that later) `book.reviews.create(username: "fun", body: "hi", rating: 5, user: user)`